cep-XXXX.md

```diff
@@ -0,0 +1,229 @@
+- *OCI tag*: A label for a specific container image within an OCI repository, identified by its `<tag>`.
+- *OCI artifact*: A container image stored in an OCI registry, specified by its OCI repository `<name>:<tag>`. Internally, an artifact consists of an OCI manifest and a set of OCI blobs (layers) that hold the artifact's data.
+- *OCI blob*: A binary blob stored in an OCI registry identified by its mediaType, digest, and size.
+- *OCI manifest*: A JSON document that describes the contents of an OCI artifact, organized as some metadata and a number of blobs. The manifest is also stored in the registry as a blob.
+
+Conda-specific terms:
+
+- *OCI-encoded package name*: The name of a conda package that is encoded to a specific format (defined below) for use in an OCI repository `<name>`. The language "encoded to OCI-form" is used below to refer to this process.
+- *OCI-encoded label/version/build string*: A conda package label, version, or build string that is encoded to a specific format (defined below) for use in an OCI image `<tag>`. The language "encoded to OCI-form" is used below to refer to this process.
+- *hashed OCI-encoded <item>*: An `<item>` that is first encoded to its OCI-form and then hashed to a specific format (defined below) for use in an OCI repository `<name>` and/or `<tag>`.
+
+For further details on the OCI-defined terms, see the full OCI specifications, the [Image Spec](https://github.com/opencontainers/image-spec) and the [Distribution Spec](https://github.com/opencontainers/distribution-spec).
+
+This specification is labeled `v1` and uses only `v1.0` of the OCI specification.
+
+### conda Packages as OCI Artifacts
+
+In order to store a conda package as an OCI artifact, we need to define how the data within the conda package is mapped into the OCI blobs, the allowed blob mediaTypes, and what additional metadata (if any) is stored in the OCI manifest.
+
+#### conda OIC Artifact Blob/Layer Media Types
+
+We define the following custom media types that MUST be used for the storage of conda packages in an OCI registry.
+
+| Blob type        | Content type              | mediaType                                   |
+|------------------|---------------------------|---------------------------------------------|
+| conda package v1 | .tar.bz2 package          | application/vnd.conda.package.v1            |
+| conda package v2 | .conda package            | application/vnd.conda.package.v2            |
+| package info     | `info` folder as gzip     | application/vnd.conda.info.v1.tar+gzip      |
+| package index    | `index.json` file         | application/vnd.conda.info.index.v1+json    |
+
+#### conda OCI Artifact Blob/Layer Structure
+
+A valid conda package as an OCI artifact MUST have the following layers:
+
+- a layer holding either the `.tar.bz2` or `.conda` package file with the mediaType `application/vnd.conda.package.v1` or `application/vnd.conda.package.v2`, respectively.
```

Comment:
Some packages (especially in defaults) are packaged as both .tar.bz2 and .conda. Can't those channels include both layers in the same artifact?

Reply:
No they cannot. conda index, the last I checked, assumed those packages hold the same information, so we should pick one and we should prefer the `.conda` one. I have added updated text to the spec.

Reply:
Yes, but they have different SHA's. With `.conda` we need to assume ZSTD decompressors are available, but I guess that's a given now.